public/application.js

```diff
@@ -4,11 +4,24 @@
 angular.module(ApplicationConfiguration.applicationModuleName, ApplicationConfiguration.applicationModuleVendorDependencies);
 
 // Setting HTML5 Location Mode
-angular.module(ApplicationConfiguration.applicationModuleName).config(['$locationProvider',
-	function($locationProvider) {
-		$locationProvider.hashPrefix('!');
-	}
-]);
+angular.module(ApplicationConfiguration.applicationModuleName).config(['$locationProvider', function($locationProvider) {
+	$locationProvider.hashPrefix('!');
+}])
+.constant('$preloadProvider', {})
+.config(['$preloadProvider', function($preloadProvider) {
+    var scripts = document.getElementsByTagName('script');
+    for (var i = 0; i < scripts.length; i++) {
+        var script = scripts[i];
+        if (script.type === 'text/preloaded') {
```

Comment:
This part seems way to easy to break if someone changes a single attribute. Any other ideas on how to handle it?